bulma-ui/src/scss/components/_badge.scss

```
@@ -158,6 +164,10 @@ $badge-animation-duration: 1.4s !default;
   content: '';
   position: absolute;
   inset: 0;
+  // Decorative halo only — never intercept clicks. pointer-events is inherited,
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)